syntax = "proto3";

message ConnectionRequest {
    string username = 1;
    string chat_id = 2;
}

message ConnectionResponse {
    bool connected = 1;
    string ip_address = 2;
    int32 port = 3;
}


service ChatService {
    rpc Connect (ConnectionRequest) returns (ConnectionResponse);
}
